eval/src/main/scala/org/dbpedia/spotlight/evaluation/EvaluateParagraphDisambiguator.scala

```diff
@@ -127,19 +134,22 @@ object EvaluateParagraphDisambiguator {
         //val default : Disambiguator = new DefaultDisambiguator(config)
         //val test : Disambiguator = new GraphCentralityDisambiguator(config)
 
-        val factory = new SpotlightFactory(config)
 
+        val db2stepdis= spotlightModel.disambiguators.get(DisambiguationPolicy.Default).disambiguator
+        db2stepdis.asInstanceOf[DBTwoStepDisambiguator].tokenizer = spotlightModel.tokenizer
         //val topics = HashMapTopicalPriorStore.fromDir(new File("data/topics"))
         val disambiguators = Set(//new TopicalDisambiguator(factory.candidateSearcher,topics),
                                  //new TopicBiasedDisambiguator(factory.candidateSearcher,factory.contextSearcher,topics)
-                                 new TwoStepDisambiguator(factory.candidateSearcher,factory.contextSearcher)
+                                 db2stepdis,
+                                 new DBBaselineDisambiguator(sfStore, resStore, candMapStore)
                                  //, new CuttingEdgeDisambiguator(factory),
                                  //new PageRankDisambiguator(factory)
                                 )
 
+
         val sources = List(//AidaCorpus.fromFile(new File("/home/pablo/eval/aida/gold/CoNLL-YAGO.tsv")),
-                           //MilneWittenCorpus.fromDirectory(new File("/home/pablo/eval/wikify/original"))
-                           PredoseCorpus.fromFile(new File("/home/pablo/eval/predose/predose_annotations.tsv")))
+                           MilneWittenCorpus.fromDirectory(new File("/home/dirk/gsoc/data/evaldata/data_gold/spotlight/wikify/original")))
```

Comment:
This file path doesn't look sane, I wonder whether we could specify the location of the gold standard in the file input or something.